app/src/main/java/com/lorenzovainigli/foodexpirationdates/view/composable/screen/SettingsScreen.kt

```diff
@@ -111,10 +121,43 @@ fun SettingsScreen(
             .padding(10.dp),
         verticalArrangement = Arrangement.spacedBy(16.dp)
     ) {
+
         Text(
             text = stringResource(R.string.behaviour),
             style = MaterialTheme.typography.labelLarge
         )
+
+        SettingsItem(
+            label = stringResource(R.string.enable_screen_protection),
+            description = stringResource(R.string.protect_your_screen)
+        ) {
+            Row(
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier.padding(start = 11.dp)
+            ) {
+                Switch(
+                    modifier = Modifier.padding(start = 4.dp),
+                    checked = isScreenProtectionEnabled,
+                    onCheckedChange = { enabled ->
+                        isScreenProtectionEnabled = enabled
+
+                        setScreenProtectionEnabled(context, enabled)
+
+                        if (enabled) {
+                            (context as Activity).window.setFlags(
+                                WindowManager.LayoutParams.FLAG_SECURE,
+                                WindowManager.LayoutParams.FLAG_SECURE
+                            )
+                            Log.d("isScreenProtectionEnabled", "Screen protection enabled: true")
+                        } else {
+                            (context as Activity).window.clearFlags(WindowManager.LayoutParams.FLAG_SECURE)
+                            Log.d("isScreenProtectionEnabled", "Screen protection enabled: false")
+                        }
+                    }
+                )
+            }
+        }
```

Comment:
Can you put this setting in a new section called `Privacy`, which is placed after `Behaviour` and before `Appearance`?